Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,46 @@
 /*  Start coding below for POSITION THE CUBES TASK*/
+.parent {
+  width: 300px;
+  height: 300px;
+  /* padding: 80px 50px;
+  padding-bottom: 80px;
+  padding: left 30px;;
+  border: 5px solid gray;
+  border-radius: 0px;*/
+  margin: 100px; 
+  background-color: #e42f68;
+}
+.child-one {
+  background: red;
+width: 100PX;
+height: 100PX;
```

Comment:
There is a semicolon (;) at the end of the margin-left property that should be removed.